qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -90,6 +90,27 @@ def _save_mount_details(self):
         self._orig_mount_details = [MountDetails(m) for m in self.mounts]
         log.info(self._orig_mount_details)
 
+    def _remove_blocklist(self):
+        # In case anything is in the OSD blocklist list, clear it out.  This is to avoid
+        # the OSD map changing in the background (due to blocklist expiry) while tests run.
+        try:
+            self.mds_cluster.mon_manager.run_cluster_cmd("osd", "blocklist", "clear")
```

Comment:
Causes:

```
2022-11-30T21:54:56.844 INFO:tasks.cephfs_test_runner:======================================================================
2022-11-30T21:54:56.845 INFO:tasks.cephfs_test_runner:ERROR: test_inject_to_empty (tasks.cephfs.test_journal_repair.TestJournalRepair)
2022-11-30T21:54:56.845 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2022-11-30T21:54:56.846 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2022-11-30T21:54:56.846 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_8b53f015ec2658258282b5e71390dde213b67dbc/qa/tasks/cephfs/cephfs_test_case.py", line 153, in setUp
2022-11-30T21:54:56.847 INFO:tasks.cephfs_test_runner:    self._remove_blocklist()
2022-11-30T21:54:56.847 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/github.com_ceph_ceph-c_8b53f015ec2658258282b5e71390dde213b67dbc/qa/tasks/cephfs/cephfs_test_case.py", line 97, in _remove_blocklist
2022-11-30T21:54:56.848 INFO:tasks.cephfs_test_runner:    self.mds_cluster.mon_manager.run_cluster_cmd("osd", "blocklist", "clear")
2022-11-30T21:54:56.848 INFO:tasks.cephfs_test_runner:TypeError: run_cluster_cmd() takes 1 positional argument but 4 were given
```

/ceph/teuthology-archive/yuriw-2022-11-30_17:19:18-fs-wip-yuri2-testing-2022-11-30-0724-pacific-distro-default-smithi/7098613/teuthology.log

(and many others)

That method's signature changed in quincy or main.

Reply:
I think it should be `raw_cluster_cmd()` instead here.

Reply:
Fixed it.